docs/guide/signals_reference.md

```diff
@@ -32,7 +32,7 @@ All Signals are relative to `eZ\Publish\Core\SignalSlot\Signal` namespace.
 |`ContentService\HideContentSignal`|`contentId`|`ContentService::hideContent()`|
 |`ContentService\PublishVersionSignal`|`contentId`</br>`versionNo`|`ContentService::publishVersion()`|
 |`ContentService\RevealContentSignal`|`contentId`|`ContentService::revealContent()`|
-|`ContentService\TranslateVersionSignal`|`contentId`</br>`versionNo`</br>`userId`|`ContentService::translationVersion()`|
+|`ContentService\TranslateVersionSignal`|`contentId`</br>`versionNo`</br>`userId`||
```

Comment:
signal should be deprecated (in 7.x) and removed in 8.x if corresponding API has been removed.

Reply:
To clarify: From what I can see `TranslateVersionSignal` is not used by kernel anymore.

So what I had in mind is that we should deprecate it in kernel in 7.5, remove it in master, and reflect the doc here for that _(mark as deprecated here in 2.5 and remove mention in master)_.

Reply:
@andrerom we should be fine with https://github.com/ezsystems/ezpublish-kernel/commit/8c46b66f3a91b3f03937f4a1448a5e02adea2ca9
Can you please re-review?

Reply:
Well we still need to add ` (deprecated)` here like done below  right?

Reply:
Also on AddTranslationInfoSignal while on it right?

Reply:
Done